deployment/eks/flyte_aws_scheduler_helm_generated.yaml

```diff
@@ -179,6 +179,8 @@ data:
       signedUrls:
         durationMinutes: 3
   storage.yaml: | 
+    propeller:
+      rawoutput-prefix: s3://<RAW_DATA_BUCKET_NAME>/data
```

Comment:
why do we need `/data` here?

Reply:
Not needed, it's the way binary handles it currently. I thought it'd be nice to align them